docs/release-notes/2.41.0.md

```diff
@@ -0,0 +1,99 @@
+
+# Percona Monitoring and Management 2.41.0
+
+
+| **Release date** | Dec 6, 2023                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+<!---
+
+!!! caution alert alert-warning "Important/Caution"
+    Crucial points that need emphasis:
+
+    - Important: A significant point that deserves emphasis.
+    - Caution: Used to mean 'Continue with care'.
+
+--->
+
+## Release Highlights
+
+### Offline metric collection
+
+The offline metric collection introduced in v2.33.0 has now been extended to QAN metrics. Previously, it was limited to Prometheus exporters only.
+
+PMM data is crucial for monitoring and troubleshooting. PMM operates on a client-server architecture, exchanging data between clients (on database servers) and servers (PMM UI interface and data storage). 
+
+Previously, if the PMM client-server connection was interrupted by a network or application issue, the system would stop collecting data, resulting in data collection interruption.
+
+Starting with PMM 2.41.0, if the PMM client and server components encounter connection problems, the QAN data gathered during these periods is stored and transmitted immediately after the connection is restored.
```

Comment:
@rasika-chivate @catalinaadam we made a decission to revert this changes, so let's remove it from the RN and docs

Reply:
Yes. Will revert it.